en/docs/install-and-setup/setup/sso/configuring-identity-server-as-external-idp-using-saml.md

```
@@ -204,18 +158,18 @@ WSO2 Identity Server acts as an identity service provider of systems enabled wit
-            Make sure your service provider configurations in the Identity Server and the identity provider configurations in API Manager are similarly reflected to each other.
+            If you configure manually, make sure your service provider configurations in the Identity Server and the identity provider configurations in API Manager are similarly reflected to each other.
 
             Example:
 
             -   If the **Response Signing Algorithm** in Identity Server is **rsa-sha256**, then the Signature Algorithm in API Manager should be **RSA with SHA256**.
             -   If you have enabled **Enable Single Logout** in the Service Provider created in the Identity Server, then you have to enable **Single Logout Profile** in the Identity Provider created in API Manager.
 
-    3.  Enable Just-in-Time Provisioning to provision the users in API Manager.
+    2.  Enable Just-in-Time Provisioning to provision the users in API Manager.
 
         [![]({{base_path}}/assets/img/setup-and-install/jit-provisioning-for-sso.png)]({{base_path}}/assets/img/setup-and-install/jit-provisioning-for-sso.png)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)